.github/workflows/build-checks.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Adjust a permission for repo
         run: |
           sudo chown -R $USER:$USER $GITHUB_WORKSPACE $HOME
-          sudo rm -rf $GITHUB_WORKSPACE/*
+          sudo rm -rf $GITHUB_WORKSPACE/* && echo "GITHUB_WORKSPACE removed" || { sleep 3 && sudo rm -rf $GITHUB_WORKSPACE/*; }
```

Comment:
How risky it'd be to actually loop till we can clean up the directory, preferably killing any process trying to access it during the iteration?

Reply:
Being occupied by another processor is just an assumption, which has not been verified. Unfortunately, the failure is not reproducible and seems an issue from infra (IBM Cloud). Even I am not sure if atomicity of figuring out a process ID and killing it is shorter than a period of time for the occupancy. I think the second try of the removal would be the best option we have without reproducing/verifying the issue.

Reply:
If your preference still goes with a loop, I will refactor the code. But the other day I luckily had a chance to face with this `directory not empty` error before. The immediate secondary removal was successful at the time.

Reply:
I'd say, just increase the sleep time to 10, for instance, to give us a little bit more leeway for now.
If this happens again, we should go for the loop approach, but I don't necessarily feel strong about taking this approach right now.

WDYT? Is it a fair compromise?